README.md

```diff
@@ -301,3 +301,99 @@ Pretty stnadard, we simply collect all the hashes and the associated types for t
 ### Evaluation
 
 As discusssed in the pinning section, evaluation is done on a fully typechecked and pinned `Expr`ession.
+
+#### Developing Inferno scripts with VScode
+
+In a shell go to /inferno and run `nix build .#vscode-inferno-lsp-server`. Then in VScode press `ctrl+shift+P` and run `Install from VSIX`. In the window, navigate to /inferno/result and select `inferno-lsp-server.vsix`.
+
+Do the same after for the VSIX created using `nix build .#vscode-inferno-syntax-highlighting`.
+
+### Regular Inferno
+
+in a shell go to /inferno and run `nix build .#inferno-lsp-server`
+run ls -al result
+copy the nix/store ... directory to your clipboard. Open VScode, press ctrl + shift + P and search for Open User Settings. Search for Inferno,
```

Comment:
Perhaps capitalize the sentence and put paths and commands inside backticks so that they are formatted as `code`?